### PR TITLE
Surface info

### PR DESCRIPTION
**Pull request by i-be-keggles**
Updated styling and layout

### Summary by CodeRabbit

**New Features:**
- Added an "About Info" section providing information about the Robotics Club, its purpose, and activities.
- Introduced a "Sideways Header" for a unique and engaging user interface.
- Enhanced the "Scoreboard" with a new "Contributions" label and an info marker button. Hovering over the info marker button reveals a hidden info box with details about GitHub commit tracking.
- Implemented a "Join" link in the header for easy club registration.

**Improvements:**
- Improved the website layout by grouping related components inside separate div elements.
- Implemented localization support, starting with the English locale.

**Removals:**
- Removed unused code for the "Matrix Background" component.

### CODE REVIEW

**Comment by i-be-keggles** [comment]
The sideways headers have some really weird horizontal alignment issues I have no idea how to fix. Maybe just remove them from the pr for now.

**Comment by velocitatem** [comment]
image
🐛

File: fe-public/src/components/Scoreboard.astro [hunk 1]
```diff
@@ -125,3 +143,21 @@ users.sort(compareUsers);
...
+    const buttons = document.querySelectorAll('button.info-marker');
+    buttons.forEach((button) => {
+        button.addEventListener('pointerenter', () => {
+            toggleInfo();
+        });
+        button.addEventListener('pointerleave', () => {
+            toggleInfo();
+        });
+    })
+</script>
```
[the remark this answers is not in the document]
Comment (vtwoptwo) [reply fe-public/src/components/Scoreboard.astro:163]: i think code rabbit knows something we dont

File: fe-public/src/pages/index.astro [hunk 2]
```diff
@@ -3,18 +3,29 @@ import Layout from '../layouts/Layout.astro';
...
+import SidewaysHeader from "../components/SidewaysHeader.astro";
 ---
 
 <Layout title="AI and Robotics Club">
     <div id="container">
         <MatrixBackground length=17 amplitude=5 period=40 falloff=2 flip="false"></MatrixBackground>
         <h1 id="header" class="text-5xl text-center h-200 top-16">AI and<br>Robotics<br>Club</h1>
-        <h2 id="header-join" class="text-3xl text-center top-32">JOIN</h2>
+        <h2 id="header-join" class="text-3xl text-center top-32">
+          <a href="https://ieconnects.ie.edu/feeds?type=club&type_id=300003041&tab=home" target="_blank">JOIN</a>
```
[the remark this answers is not in the document]
Comment (velocitatem) [reply fe-public/src/pages/index.astro]: 👍🏻  fe-public/src/scripts/locales.json

**Review by velocitatem — APPROVED** [review]
🚀

**Pull request merged**